deepchem/feat/molecule_featurizers/molgan_featurizer.py

```diff
@@ -0,0 +1,214 @@
+import logging
```

Comment:
Can you create unit tests for `GraphMatrix` and for `MolGanFeaturizer`? A few tests of the featurize/defeaturize methods on basic smiles strings to test the featurization and inversion would be great :)

Reply:
Will work on that

Reply:
Do you think GraphMatrix class should be in a different place?

Reply:
I think it's probably fine to have `GraphMatrix` in this file since we don't use it anywhere else in the codebase yet

Reply:
I have updated unit tests, see if they are enough or need more. For featurizer I have created two tests to see if forward/backword conversion works both on smiles and rdkit.Mols